WordPress/src/main/java/org/wordpress/android/util/config/AppConfig.kt

```diff
@@ -90,10 +90,15 @@ class AppConfig
     }
 
     sealed class FeatureState(open val isEnabled: Boolean, val name: String) {
-        data class ManuallyOverriden(override val isEnabled: Boolean) : FeatureState(isEnabled, "manually_overriden")
-        data class BuildConfigValue(override val isEnabled: Boolean) : FeatureState(isEnabled, "build_config_value")
-        data class RemoteValue(override val isEnabled: Boolean) : FeatureState(isEnabled, "remote_source_value")
-        data class StaticValue(override val isEnabled: Boolean) : FeatureState(isEnabled, "static_source_value")
-        data class DefaultValue(override val isEnabled: Boolean) : FeatureState(isEnabled, "default_source_value")
+        data class ManuallyOverriden(override val isEnabled: Boolean) :
+                FeatureState(isEnabled, "manually_overriden")
+        data class BuildConfigValue(override val isEnabled: Boolean) :
+                FeatureState(isEnabled, "build_config_value")
+        data class RemoteValue(override val isEnabled: Boolean) :
+                FeatureState(isEnabled, "remote_source_value")
+        data class StaticValue(override val isEnabled: Boolean) :
+                FeatureState(isEnabled, "static_source_value")
+        data class DefaultValue(override val isEnabled: Boolean) :
+                FeatureState(isEnabled, "default_source_value")
```

Comment:
Suggestion (💡): In order to make the above adhere to the max line length restriction, I would have updated the `FeatureState` sealed class and its data classes like that below:

```
    sealed class FeatureState(open val isEnabled: Boolean, val name: String) {
        data class ManuallyOverriden(
            override val isEnabled: Boolean
        ) : FeatureState(isEnabled, "manually_overriden")
        data class BuildConfigValue(
            override val isEnabled: Boolean
        ) : FeatureState(isEnabled, "build_config_value")
        data class RemoteValue(
            override val isEnabled: Boolean
        ) : FeatureState(isEnabled, "remote_source_value")
        data class StaticValue(
            override val isEnabled: Boolean
        ) : FeatureState(isEnabled, "static_source_value")
        data class DefaultValue(
            override val isEnabled: Boolean
        ) : FeatureState(isEnabled, "default_source_value")
    }
```

I would have done the above because it is the recommended way for data classes, that is, to have their field on separate lines. Wdyt? 🤔

Reply:
👋 @Hoossayn and thank you for updating this PR with this [extra commit](https://github.com/wordpress-mobile/WordPress-Android/pull/17070/commits/72ffd58d3fc38248b07a04aca9feb92b1576528c) that addresses this comment.

Although you did update it as per my suggestion (thank you), I am seeing that the formatting is now incorrect. Would you mind fixing the formatting on all those data classes? For example:

- ❌ This formatting is incorrect:
```
        data class ManuallyOverriden(
            override val isEnabled: Boolean
            ) : FeatureState(isEnabled, "manually_overriden")
```
- ✅ This formatting is correct:
```
        data class ManuallyOverriden(
            override val isEnabled: Boolean
        ) : FeatureState(isEnabled, "manually_overriden")
```

PS: On your IDE (`Android Studio`) you could also run `Reformat Code` and see it automatically correcting these formatting issues I mentioned above.